plugins/app-backend/src/service/router.ts

```diff
@@ -299,7 +298,9 @@ async function createEntryPointRouter({
   if (staticFallbackHandler) {
     staticRouter.use(staticFallbackHandler);
   }
-  staticRouter.use(notFoundHandler());
+  staticRouter.use((_req: Request, res: Response) => {
+    res.status(404).end();
```

Comment:
Should we use `MiddlewareFactory.notfound` by convention? I wondered if it would be useful to have a `createTestRouter` or something that installs the standard middlewares by default... Nothing to block this pull request, just a possible idea of ​​follow-up 🙂 .

Reply:
Here it turned out to be complex to keep the middleware factory version. I honestly forget why but I had it here first and then removed it. Probably because of the dependency change needed.